inscriptions/ISic003323.xml

```diff
@@ -191,108 +196,35 @@
-                        <ex>ua</ex>
-                    </expan>
-                    <expan>
-                        <abbr>p</abbr>
-                        <ex>ecunia</ex>
-                    </expan>
-                    .
-                </ab>
-                <!--This EDR text 033622 originally came via the Europeana EAGLE project, http://www.eagle-network.eu/-->
-            </div>
-            <div type="edition" xml:space="preserve" subtype="PHI" n="PHI313633">
+            <div type="edition" xml:space="preserve" xml:lang="la" resp="#DEE">
                 <ab>
-                    <lb n="1"/>
-                    <supplied reason="lost">— — — — — — — —</supplied>io <supplied reason="lost">I</supplied>nnocenti
-                    <lb n="2"/>
-                    <supplied reason="lost">— — — — — — — — — —</supplied>․V
-                    <supplied reason="lost">— — — — — — — —</supplied>
-                    <lb n="3"/>
-                    <supplied reason="lost">— — —</supplied>․․<supplied reason="lost">—</supplied>․N
-                    <unclear>IA</unclear>
-                    <supplied reason="lost">—</supplied>NIS ∙ HV<supplied reason="lost">— — —</supplied> Obito ∙ ex
-                    <lb n="4"/>
-                    <supplied reason="lost">(?)testamento eius os</supplied>sa h<supplied reason="lost">eic in cella
-                    absi</supplied>data ∙ sunt ∙<lb n="5"/>E
-                    <supplied reason="lost">— — — — — — — — — —</supplied>
-                    <unclear>LI</unclear>
-                    <expan>
-                        <abbr>eor</abbr>
-                        <ex>um</ex>
-                    </expan>
-                    statuae<lb n="6"/>․<supplied reason="lost">— — — et statua seden</supplied>s ∙ eius permissum est
-                    <lb n="7"/>
-                    <supplied reason="lost">— — — — — — — — — —</supplied>․IAMCAS
-                    <expan>
-                        <abbr>mat</abbr>
-                        <ex>er</ex>
-                    </expan>
-                    ∙
-                    <expan>
-                        <abbr>s</abbr>
-                        <ex>ua</ex>
-                    </expan>
-                    <expan>
-                        <abbr>p</abbr>
-                        <ex>ecunia</ex>
-                    </expan>
-                    .
-                    <lb n="8"/>
+                    <lb n="1"/><gap reason="lost" extent="unknown" unit="character"/><persName type="attested"><name type="nomen"><seg part="F"><unclear>i</unclear><unclear>o</unclear></seg></name> <name type="cognomen"><supplied reason="lost">I</supplied><unclear>n</unclear><unclear>n</unclear><unclear>o</unclear><unclear>c</unclear>enti</name></persName>
+                    <lb n="2"/><gap reason="lost" extent="unknown" unit="character"/><gap reason="illegible" quantity="1" unit="character"/>V <gap reason="lost" extent="unknown" unit="character"/>
```

Comment:
All capitalised uninterpretable bits of words should go in `<orig>`, like 'V' on this line and 'NIA' etc. on the next line.